src/main/java/mate/academy/validation/password/FieldMatcher.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.validation.password;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import java.util.Objects;
+import org.springframework.beans.BeanWrapperImpl;
+
+public class FieldMatcher implements ConstraintValidator<FieldMatch, Object> {
+    private String field;
+    private String fieldMatch;
+
+    @Override
+    public void initialize(FieldMatch constraintAnnotation) {
+        this.field = constraintAnnotation.field();
+        this.fieldMatch = constraintAnnotation.fieldMatch();
+    }
+
+    @Override
+    public boolean isValid(Object value, ConstraintValidatorContext constraintValidatorContext) {
+        Object field = new BeanWrapperImpl(value).getPropertyValue(this.field);
+        Object fieldMatch = new BeanWrapperImpl(value).getPropertyValue(this.fieldMatch);
+        return Objects.equals(field, fieldMatch) && !((String) field).isBlank();
```

Comment:
you should not check for blankness here

instead `password` and `repeatPassword` should be `@NotBlank`

Reply:
done